packages/CollapsibleText/README.md

```diff
@@ -0,0 +1,38 @@
+## CollapsibleText
```

Comment:
I'm 👍 but would `CollapsibleContent` be a better name?

Reply:
We could squirrel all day long on a name... but to me `content` could suggest _anything_ whereas `text` at least _suggests_ it's intended only for static elements (plain or rich text).  

I'd also suggested `<CollapsiblePreview>` and `<CollapsibleTeaser>`... but somewhat preferred JP's suggestion or `<CollapsibleText>`, even if it is kinda boring 😆.